Tests/XCUITests/FirstRunTourTests.swift

```diff
@@ -55,23 +56,36 @@ class FirstRunTourTests: BaseTestCase {
         waitForExistence(topSites)
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2250844
     func testCloseTour() {
         app.buttons["\(AccessibilityIdentifiers.Onboarding.closeButton)"].tap()
         let topSites = app.collectionViews.cells[AccessibilityIdentifiers.FirefoxHomepage.TopSites.itemCell]
         waitForExistence(topSites)
     }
 
-    // MARK: Private
-    private func goToNextScreen() {
-        waitForExistence(app.buttons["\(rootA11yId)PrimaryButton"], timeout: 10)
-        app.buttons["\(rootA11yId)PrimaryButton"].tap()
-        currentScreen += 1
-    }
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/67227
+    func testWhatsNewPage() {
```

Comment:
I see this test enabled in the Smoketest but it is marked just as Functional... we should disable in the SmokeXCUItest test plan and enable in the Full Functional instead